src/mesh/generated/mesh.pb.h

```diff
@@ -70,7 +70,9 @@ typedef enum _HardwareModel {
     /* Custom Disaster Radio esp32 v3 device https://github.com/sudomesh/disaster-radio/tree/master/hardware/board_esp32_v3 */
     HardwareModel_DR_DEV = 43, 
     /* M5 esp32 based MCU modules with enclosure, TFT and LORA Shields. All Variants (Basic, Core, Fire, Core2, Paper) https://m5stack.com/ */
-    HardwareModel_M5STACK = 44, 
+    HardwareModel_M5STACK = 44,
+    /* B&Q Consulting Station Edition G1: https://uniteng.com/wiki/doku.php?id=meshtastic:station */
+    HardwareModel_STATION_G1 = 45,  
```

Comment:
This file is auto-generated from the protobuf definitions. Can you please remove it from the PR? I just approved the change on the other repository, so this will reappear soon :-)

Reply:
Learned something new, and modified the code.